tests/benchmarks/test_quantum_volume.py

```diff
@@ -0,0 +1,26 @@
+from metriq_gym.benchmarks.quantum_volume import calc_stats, QuantumVolumeData
+
+
+def test_calc_stats():
+    job_data = QuantumVolumeData(
+        provider_job_id="test_job_id",
+        qubits=2,
+        shots=100,
+        depth=2,
+        confidence_level=0.7,
+        ideal_probs=[
+            {0: 0.8, 1: 0.2},
+            {0: 0.6, 1: 0.4},
+        ],
+        trials=2,
+    )
+    counts = [
+        {"0": 80, "1": 20},
+        {"0": 60, "1": 40},
+    ]
+    stats = calc_stats(job_data, counts)
+    assert stats.confidence_pass is True  # All trials pass confidence level
+
+    job_data.confidence_level = 0.9
```

Comment:
Based on the way in which the `p_val` and `confidence_pass` are defined, even with this increase in confidence level, it will only go from 0 to something slightly larger than zero (something like 0.02...). So IOW, the way it's currently defined will always yield a confidence pass (not a failure).

Reply:
IBM seems to calculate a "confidence pass" based on the p-value for the hypothesis test to reject that the data came from a device with less than 2/3 heavy-output generation rate. Instead, we define a "confidence pass" based on the p-value for the hypothesis test to reject that the data came from a device with 1/2 heavy-output generation rate, i.e. from uniform random guessing. However, I believe that IBM's p-value can _always_ be simply converted to ours and vice versa, without loss of information. To satisfy IBM's definition, it could be sufficient that we set our confidence threshold very high, but it works, either way.